irida_import/config.py

```diff
@@ -127,6 +127,17 @@ def _load_from_file(self):
             self.CLIENT_ID = config.get('IRIDA', 'client_id')
             self.CLIENT_SECRET = config.get('IRIDA', 'client_secret')
 
+            try:
+                self.IRIDA_STORAGE_TYPE = config.get('IRIDA', 'irida_storage_type')
+            except:
+                self.IRIDA_STORAGE_TYPE = 'local'
+
+            self.AZURE_ACCOUNT_NAME = config.get('IRIDA', 'azure_account_name')
```

Comment:
AZURE and AWS options should only be tried to parsed if IRIDA_STORAGE type is azure or aws respectively.

Reply:
Updated in 50e1f7291bd06d1da1f8fe3f432dacba6314d133